frontend/text.js

```diff
@@ -137,6 +137,9 @@ class Text {
    * Inserts new list items `values` starting at position `index`.
    */
   insertAt(index, ...values) {
+    if (typeof values[0] === 'string' && values[0].length > 1) {
+      values = values[0].split("")
```

Comment:
This destroys surrogate pairs. See #334. You should use `values = [...values[0]]`

Reply:
Thanks for the comment. I fixed it and and added a test in dfe1929